### PR TITLE
Added a new maintainer to the list: @povilasv welcome!

### PR DESCRIPTION
**Pull request by bwplotka**
Signed-off-by: Bartek Plotka <bwplotka@gmail.com>

### CODE REVIEW

**Comment by bwplotka** [comment]
Please review carefully add the mention below table (:

File: MAINTAINERS.md [hunk 1]
```diff
@@ -5,7 +5,18 @@
...
 | Frederic Branczyk     | fbranczyk@gmail.com    | `@brancz`                | [@brancz](https://github.com/brancz)                       |
-| Giedrius Statkevičius | giedriuswork@gmail.com | `@Giedrius Statkevičius` | [@GiedriusS](https://github.com/GiedriusS) |
+| Giedrius Statkevičius | giedriuswork@gmail.com | `@Giedrius Statkevičius` | [@GiedriusS](https://github.com/GiedriusS)                 |
+| Povilas Versockas     | p.versockas@gmail.com  | `@povilasv`              | [@povilasv](https://github.com/povilasv)
+
+We are bunch of people from different companies with various interests and skills.
+We are from different parts of Europe: Germany, Lithuania, Poland and UK. 
+We have something in common though: We all share the love for OpenSource, Golang, Prometheus and Observability topics.
+
+As either Software Developers or SRE (or both!) we've chosen to maintain (mostly in our free time) the Thanos, defacto way to scale awesome [Prometheus](https://prometheus.io) project.
```
Comment (improbable-ludwik) [inline MAINTAINERS.md]: `we've chosen to maintain (mostly in our free time) the Thanos, defacto way to scale awesome [Prometheus](https://prometheus.io) project.`
->
`we've chosen to maintain (mostly in our free time) Thanos, the de facto way to scale the awesome [Prometheus](https://prometheus.io) project.`

File: MAINTAINERS.md [hunk 2]
```diff
@@ -5,7 +5,18 @@
...
+| Giedrius Statkevičius | giedriuswork@gmail.com | `@Giedrius Statkevičius` | [@GiedriusS](https://github.com/GiedriusS)                 |
+| Povilas Versockas     | p.versockas@gmail.com  | `@povilasv`              | [@povilasv](https://github.com/povilasv)
+
+We are bunch of people from different companies with various interests and skills.
+We are from different parts of Europe: Germany, Lithuania, Poland and UK. 
+We have something in common though: We all share the love for OpenSource, Golang, Prometheus and Observability topics.
+
+As either Software Developers or SRE (or both!) we've chosen to maintain (mostly in our free time) Thanos, the de facto way to scale awesome [Prometheus](https://prometheus.io) project.
+
+Feel free to contact us (preferably on Slack) anytime for feedback, questions or beers :beers:
```
Comment (brancz) [inline MAINTAINERS.md]: I like :coffee: and :tea: as well :)

Comment (bwplotka) [reply MAINTAINERS.md]: I would forget about the important stuff!!

**Pull request merged**